crates/ruff_diagnostics/src/edit.rs

```diff
@@ -4,13 +4,13 @@ use serde::{Deserialize, Serialize};
 
 #[derive(Clone, Debug, PartialEq, Eq, PartialOrd, Ord)]
 #[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
-pub struct Fix {
+pub struct Edit {
```

Comment:
Do you want to use this change as an opportunity to document the struct?

Reply:
Why would we want docs?

Reply:
🧐

Reply:
:joy: